src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -100,28 +102,45 @@ open class SmkFileReference(
         return null
     }
 
-
-    private fun findVirtualFile(): VirtualFile? {
+    private fun findVirtualFile(searchFromRoot: Boolean = false): VirtualFile? {
         val stringLiteral = element.argumentList?.arguments?.firstOrNull {
             it is PyStringLiteralExpression && it.stringValue == path
         } ?: return null
 
-        val parentFolder = element.containingFile.originalFile.virtualFile?.parent ?: return null
-
-        val filePath = (stringLiteral as? PyStringLiteralExpression)?.stringValue
+        val filePath = (stringLiteral as? PyStringLiteralExpression)?.stringValue!!
         val vfm = VirtualFileManager.getInstance()
-
+        if (searchFromRoot) {
+            //search in all content roots
+            val contentRoots = ProjectRootManager.getInstance(element.project).contentRoots ?: return null
```

Comment:
Accoding to contentRoots signature cannot be null, see ProjectRootManager class:
```
public abstract VirtualFile @NotNull [] getContentRoots();
```
So null check not needed